editor.planx.uk/src/pages/Team/components/RenameDialog.tsx

```diff
@@ -0,0 +1,163 @@
+import { isApolloError } from "@apollo/client";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import Typography from "@mui/material/Typography";
+import { Form, Formik, FormikConfig } from "formik";
+import gql from "graphql-tag";
+import { useToast } from "hooks/useToast";
+import { client } from "lib/graphql";
+import React from "react";
+import { URLPrefix } from "ui/editor/URLPrefix";
+import InputLabel from "ui/public/InputLabel";
+import Input from "ui/shared/Input/Input";
+import { slugify } from "utils";
+import { object, string } from "yup";
+
+interface RenameFlow {
+  name: string;
+  slug: string;
+}
+
+interface Props {
+  isDialogOpen: boolean;
+  handleClose: () => void;
+  flow: {
+    name: string,
+    slug: string,
+    id: string,
+  }
+}
+
+const validationSchema = object().shape({
+  slug: string().required("Slug is required"),
+  name: string().required("Name is required")
+});
```

Comment:
nit: do these schemas also need `.trim()` to trigger error on empty strings?

Can currently enter an empty string and click "Rename" and it just sort of freezes, no visible error
![Screenshot from 2025-05-08 16-54-13](https://github.com/user-attachments/assets/d72b2c33-3276-42d1-a48a-fb7bb9eca2ea)